### DISCUSSION TITLE
Discount vs Vouchers?

### DISCUSSION THREAD

**Discussion by jha-r** [saleor/saleor#16308] [Q&A] [answered]
I'm using Saleor 3.19, and I am not able to figure out what the difference is between Discounts and Vouchers.

On the dashboard->Discounts, new discounts can be created. And when I apply the same discount it thorws error 'Invalid Promo Code'

On the other hand, Vouchers work. Not sure what I'm missing?

![image](https://github.com/saleor/saleor/assets/57193136/26445e29-e339-4250-9def-731a02937217)

![image](https://github.com/saleor/saleor/assets/57193136/cd3d8670-58fc-47a7-b2fa-3e9d070faa8b)

![image](https://github.com/saleor/saleor/assets/57193136/845943ce-0327-4500-bd67-5bec25d81502)

**Comment by timuric** [comment, marked as answer]
@jha-r Discounts are catalog level, which does not need a promo code and can be available to anyone shopping in the corresponding channel. (the name of the discount is not related to a promo code)
You can learn more about promotions and vouchers https://docs.saleor.io/docs/developer/discounts/overview
